src/components/ose/EditQuota.vue

```diff
@@ -25,7 +25,7 @@
                 </b-input>
             </b-field>
 
-            <b-field label="Neue Memory Quotas [GB]"
+            <b-field label="New memory quotas [GB]"
```

Comment:
```suggestion
            <b-field label="New memory quota [GB]"
```
Wie oben, kein Plural.